### ISSUE TITLE
Key behavior in Browser

### ISSUE THREAD

**Issue by tonypdmtr** [sqlitebrowser/sqlitebrowser#483]
When searching, the list of found items appears in the grid below.

Pressing down arrow (while still in the search box) highlights row number but not the whole record (as if when clicked).

Pressing ESC, when in the result grid, could go back up to the search box to let one type another search.  Now it does nothing.

Thanks.

**Comment by justinclift** [comment]
That's a bit confusing.  Are you meaning this is a regression in behaviour?  eg it used to do things ok, but now its not

Or are you suggesting a newn improved behaviour? :smile:

**Comment by tonypdmtr** [comment]
No, I just noticed this.  It's not a bug, rather a 'polishing' of the UI for easier use by keyboard.

**Comment by justinclift** [comment]
Cool. :smile:
